src/keyboard.rs

```diff
@@ -11,7 +53,32 @@ use dq::DeviceQuery;
 
 const KEY_EVENT_FOCUS_DELAY_MS: u64 = 20;
 
-
+/// The keys detectable by `ruscii`.
+///
+/// A value, [`Key::Unknown`], is provided when a key is detected but the type of key cannot be
+/// ascertained.
+///
+/// ## Exceptional Behavior
+///
+/// Certain values of this enum represent _positions_ on the keyboard rather than the key type.
+/// These values might behave differently than expected when using different keyboard layouts.
+/// This includes:
+///
+/// - grave/backtick `` ` ``
+/// - minus `-`
+/// - equal `=`
+/// - left and right bracket `[]`
+/// - forward and back slash `/\ `
+/// - semicolon `;`
+/// - apostrophe `'`
+/// - comma `,`
+/// - dot `.`
+///
+/// These keys are named according to their function in a U.S. ASCII keyboard layout. Differing
+/// layouts may vary in which key event is fired.
+///
+/// A differing layout may also affect how key events are fired; certain keyboards may generate
+/// only one event by pressing two keys.
```

Comment:
Yes... this is something that depends on how the OS interprets the `stdin`. My advice here is to let the user customize their keys, `keys.config` file that the app can read with some sane defaults. Thanks for explaining that thing in the docs!